.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
{
  "name": "25-angular_auth",
  "version": "1.0.0",
  "description": "![cf](https://i.imgur.com/7v5ASc8.png) Lab 25 - Client Side Auth ======",
  "main": "karma.conf.js",
  "scripts": {
    "build": "./node_modules/webpack/bin/webpack.js",
    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot",
    "lint": "./node_modules/eslint/bin/eslint.js ./**/*.js"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/darms/25-angular_auth.git"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/darms/25-angular_auth/issues"
  },
  "homepage": "https://github.com/darms/25-angular_auth#readme",
  "dependencies": {
    "angular": "^1.6.3",
    "angular-animate": "^1.6.3",
    "angular-touch": "^1.6.3",
    "angular-route": "^1.6.3",
    "angular-ui-router": "^0.4.2",
    "babel-core": "^6.24.0",
    "babel-loader": "^6.4.1",
    "babel-preset-es2015": "^6.24.0",
    "camelcase": "^4.0.0",
    "clean-webpack-plugin": "^0.1.16",
    "css-loader": "^0.27.3",
    "dotenv": "^4.0.0",
    "extract-text-webpack-plugin": "^2.1.0",
    "html-webpack-plugin": "^2.28.0",
    "file-loader": "^0.10.1",
    "html-loader": "^0.4.5",
    "node-sass": "^4.5.1",
    "pascalcase": "^0.1.1",
    "resolve-url-loader": "^2.0.2",
    "sass-loader": "^6.0.3",
    "style-loader": "^0.16.1",
    "ui-router": "^1.0.0-alpha.3",
    "url-loader": "^0.5.8",
    "webpack": "^2.3.2"
  },
  "devDependencies": {
    "angular-mocks": "^1.6.3",
    "eslint": "^3.18.0",
    "jasmine-core": "^2.5.2",
    "karma": "^1.5.0",
    "karma-jasmine": "^1.1.0",
    "karma-mocha-reporter": "^2.2.3",
    "karma-phantomjs-launcher": "^1.0.4",
    "karma-webpack": "^2.0.3",
    "webpack-dev-server": "^2.4.2"
  }
}
    //this is actually geared towards your package.json, you are missing some dependencies